python-runnables/create-api-service-v2/runnable.json

```diff
@@ -58,11 +58,57 @@
             "label": "Code environment",
             "type": "SEPARATOR"
         },
+        {
+            "name": "code_env_options",
+            "label": "Code environment options",
+            "type": "SELECT",
+            "description":"Creation of code environment is only available for admin.",
```

Comment:
I think this description should be removed, it creates confusion to users when they don't see the `create new environment` option

Reply:
Fixed in a331294. Is it enough to be understandable? I prefer leaving something about being an admin to create a codeenv, that way, the user will now he has to ask his admin to do this.

Reply:
Totally removed description in 447bdeb